### ISSUE TITLE
PlatformException(no_window, The active window does not exist, null, null)

### ISSUE THREAD

**Issue by Verzu23** [zonble/flutter_window_close#8]
Hello guys, I'm trying to execute the example but i get this error every time i click the close button.

`[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: PlatformException(no_window, The active window does not exist, null, null)
#0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:652:7)
#1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:310:18)
<asynchronous suspension>
`

Do you have any suggestion? 

Thanks a lot

**Comment by maurovanetti** [comment]
I guess this is due to #7

**Comment by Hasan-Amkieh** [comment]
you can fix the issue by changing the C++ function named "RegisterWithRegistrar" with these few lines, what it does is basically finding the handle of the window using the name of the window, 

replace this line:

```C++
HWND handle = GetActiveWindow();
```

with these lines, replace mars_nav with the window name of your app:

```C++
char windowName[] = "mars_nav";
wchar_t wtext[20];
size_t convertedChars = 0;
mbstowcs_s(&convertedChars, wtext, 20, windowName, strlen(windowName) + 1);
LPWSTR ptr = wtext;

HWND handle = FindWindowW(nullptr, ptr);
```


I hope this fixes your issue, 
Best regards,

**Comment by zonble** [comment]
Should be fixed in #4.
